packages/design-tokens/tokens/color.json

```diff
@@ -1,4 +1,230 @@
+      "500-rgb-id": "--color-red-500-rgb",
+      "500-id": "--color-red-500",
+      "600-rgb": "var(--color-red-600-rgb, 176, 33, 48)",
+      "600-rgb-id": "--color-red-600-rgb",
+      "600-id": "--color-red-600",
+      "700-rgb": "var(--color-red-700-rgb, 108, 30, 32)",
+      "700-rgb-id": "--color-red-700-rgb",
+      "700-id": "--color-red-700"
+    },
+    "orange": {
+      "100": "var(--color-orange-100, #FFF1EE)",
+      "200": "var(--color-orange-200, #FDD6CD)",
+      "300": "var(--color-orange-300, #FDBAAB)",
+      "400": "var(--color-orange-400, #FC9E8A)",
+      "500": "var(--color-orange-500, #FA7558)",
+      "600": "var(--color-orange-600, #B85D4A)",
+      "700": "var(--color-orange-700, #903c00)",
+      "100-rgb": "var(--color-orange-100-rgb, 255, 241, 238)",
+      "100-rgb-id": "--color-orange-100-rgb",
+      "100-id": "--color-orange-100",
+      "200-rgb": "var(--color-orange-200-rgb, 253, 214, 205)",
+      "200-rgb-id": "--color-orange-200-rgb",
+      "200-id": "--color-orange-200",
+      "300-rgb": "var(--color-orange-300-rgb, 253, 186, 171)",
+      "300-rgb-id": "--color-orange-300-rgb",
+      "300-id": "--color-orange-300",
+      "400-rgb": "var(--color-orange-400-rgb, 252, 158, 138)",
+      "400-rgb-id": "--color-orange-400-rgb",
+      "400-id": "--color-orange-400",
+      "500-rgb": "var(--color-orange-500-rgb, 250, 117, 88)",
+      "500-rgb-id": "--color-orange-500-rgb",
+      "500-id": "--color-orange-500",
+      "600-rgb": "var(--color-orange-600-rgb, 184, 93, 74)",
+      "600-rgb-id": "--color-orange-600-rgb",
+      "600-id": "--color-orange-600",
+      "700-rgb": "var(--color-orange-700-rgb, 144, 60, 0)",
+      "700-rgb-id": "--color-orange-700-rgb",
+      "700-id": "--color-orange-700"
+    },
+    "gray": {
+      "100": "var(--color-gray-100, #F9F9F9)",
+      "200": "var(--color-gray-200, #F4F4F5)",
+      "300": "var(--color-gray-300, #EAEAEC)",
+      "400": "var(--color-gray-400, #CDCDD0)",
+      "500": "var(--color-gray-500, #8C8C97)",
+      "600": "var(--color-gray-600, #524E56)",
+      "100-rgb": "var(--color-gray-100-rgb, 249, 249, 249)",
+      "100-rgb-id": "--color-gray-100-rgb",
+      "100-id": "--color-gray-100",
+      "200-rgb": "var(--color-gray-200-rgb, 244, 244, 245)",
+      "200-rgb-id": "--color-gray-200-rgb",
+      "200-id": "--color-gray-200",
+      "300-rgb": "var(--color-gray-300-rgb, 234, 234, 236)",
+      "300-rgb-id": "--color-gray-300-rgb",
+      "300-id": "--color-gray-300",
+      "400-rgb": "var(--color-gray-400-rgb, 205, 205, 208)",
+      "400-rgb-id": "--color-gray-400-rgb",
+      "400-id": "--color-gray-400",
+      "500-rgb": "var(--color-gray-500-rgb, 140, 140, 151)",
+      "500-rgb-id": "--color-gray-500-rgb",
+      "500-id": "--color-gray-500",
+      "600-rgb": "var(--color-gray-600-rgb, 82, 78, 86)",
+      "600-rgb-id": "--color-gray-600-rgb",
+      "600-id": "--color-gray-600"
+    },
+    "white": "var(--color-white, #FFFFFF)",
```

Comment:
/meganitpick The different cases in the hex strings irk me a little - the generated strings are all lowercase. I believe one of our stylelint rules is opinionated about whether we prefer caps or not in hex strings.

Reply:
nah thats not a nitpick i agree! good to do it now